backend/question-service/src/lib/validators/UpdateQuestionValidator.ts

```diff
@@ -20,22 +20,14 @@ export const UpdateQuestionValidator = z.object({
   examples: z
     .array(
       z.object({
-        input: z.string().min(1).max(1000),
+        input: z.string().min(1).max(10000),
```

Comment:
will these limits be able to accommodate images? i think last time i tried to insert an image it was 50k characters 😓

Reply:
I don't think input needs to contain image, jus the explanation will have the image right? I increased this size so that it can include all the database tables.

Reply:
👌🏻

Reply:
this is only for the examples field, the images with explanation should reside in the description field for now.
basically, this example field is more for inputting test cases for code execution alr......

Reply:
According to one of the questions shown in the Assignment pdf, the examples field can have image also:
![image](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g05/assets/69421979/4790ca5b-1b74-4477-8156-99d46f172b36)
To be honest, I didn't prepare my code to render this kind of situation haha. So I was thinking we simply put the image and the text explanation inside the `explanation` field for now, if we really have this kind of question.

> this example field is more for inputting test cases for code execution alr

What do you mean by this? The way I see examples is that it is just giving visual examples for the user to better understand the problem. It is not the test case, although we can set the test case to be the same as the examples. Regarding renaming to test cases, I will suggest we don't do it for now, since I believe we don't have plans to integrate with test cases right??

Reply:
the html parse helps to render images in question description already, so I don't think that will be an issue.

Having an extra explanation field is definitely better, but in terms of effort, I don't think it is justifiable.
Cos how do you want to handle the images in that field? 
easiest is to ask the user to key in the external image url and store it as a string. The user experience will not be good, no point in doing that.
the second option is to use a rich text editor like the description field. which is feasible but complicates the form. 

I guess since it is not an essential feature, I suggest we ask the user to key in examples with images and explanations all in the description field and let the html parser handle them together.